### ISSUE TITLE
No issues checkmark on Project Info Cards should have label/alt text

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1169]
The No Issues checkmark on the Project Info cards in /administrator view should have aria-label or alt-text

**Comment by sudo-may** [comment]
Decided it would be better to exclude checkmark since there is already text provided

**Comment by sudo-may** [comment]
PR: https://github.com/NationalSecurityAgency/skills-service/pull/1338

**Comment by rmmayo** [comment]
:+1:
